src/portals/blockchain-frontend/apps/blockchain-frontend/components/courses-container/CoursesContainer.tsx

```diff
@@ -56,7 +57,10 @@ export default function CoursesContainer() {
       },
     },
   ];
-
+  useFetchCourseEffect(fetchCourses)
```

Comment:
don't need to use this here. Fetch courses in `state.ts` and just get the result to this file.

Reply:
fixed